File: src/main/java/constants/ServerConstants.java
package constants;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;

public class ServerConstants {

    public static final short VERSION = 95;
    public static final int CHANNELS = 3;
    public static final int CHANNEL_LOAD = 100;
    public static final String IP = "25.55.234.58";
    public static final int LOGIN_PORT = 8484;
    public static final boolean LOG = true;

    public static final String DB_USER = "root";
    public static final String DB_PASS = "";
    public static final String DB_URL = "jdbc:mysql://localhost:3306/mao?serverTimezone=UTC";

    public static List<List<String>> COMMAND_LIST = new ArrayList<>();
    public static HashMap<String, String> COMMAND_FILE_LIST = new HashMap<String, String>();
}

File: src/main/java/constants/ServerConstants.kt
package constants

object ServerConstants {

    const val VERSION: Short = 95
    const val CHANNELS = 3
    const val IP = "217.62.201.166"
    const val LOG = true
    const val DEBUG = true // in some cases a different action should be taken during server development in order to prevent some annoying stuff from happening

    const val DB_USER = "root"
    const val DB_PASS = ""
    const val DB_URL = "jdbc:mysql://localhost:3306/mao?serverTimezone=UTC"

    const val SCRIPTS_ROOT = "scripting.scripts"

    const val RANKING_TIMER: Long = 10 /*minutes*/ * 60 * 1000
    const val RESPAWN_TIMER: Long = 5 /*seconds*/ * 1000
    const val DROP_CLEAR_TIMER: Long = 10 /*seconds*/ * 1000
    const val BULK_DUMP_TIMER: Long = 5 /*minutes*/ * 60 * 1000
}